novuhq/website
# chore: deps update, migrate to esm

This pull request includes dependencies update.

Steps to check:
- open [preview](https://deploy-preview-134--novu-website.netlify.app/), open development mode
- make sure that everything looks and works as expected
